src/creature.cpp

```diff
@@ -1314,7 +1309,7 @@ int64_t Creature::getStepDuration() const
 		calculatedStepSpeed = 1;
 	}
 
-	Item* ground = tile->getGround();
+	auto ground = tile->getGround();
```

Comment:
Could this be `const auto`
```suggestion
	const auto ground = tile->getGround();
```